aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -258,6 +258,7 @@ IMPLEMENT_FLOAT_KERNEL(FLOATING, cos)
 // IMPLEMENT_FLOAT_KERNEL(FLOATING, cosh)
 IMPLEMENT_FLOAT_KERNEL(FLOATING, erf)
 IMPLEMENT_FLOAT_KERNEL(FLOATING, erfc)
+IMPLEMENT_FLOAT_KERNEL(FLOATING, erfinv)
```

Comment:
I prefer if we clearly state that erfinv doesn't support AVX2 vectorization (dispatch it via `cpu_loop`). That might even make code faster, and definitely will generate smaller kernels.

Reply:
> Please remove `TH_erfinv` and usage of it.

Done